Modules/Quotas/src/Quotas.Domain/Aggregates/Identities/Identity.cs

```diff
@@ -21,4 +22,43 @@ public void AssignTierQuotaFromDefinition(TierQuotaDefinition definition)
         var tierQuota = new TierQuota(definition, Address);
         _tierQuotas.Add(tierQuota);
     }
+
+    public async Task UpdateMetrics(IEnumerable<string> metrics, IMetricCalculatorFactory factory, CancellationToken cancellationToken)
```

Comment:
```suggestion
    public async Task UpdateMetrics(IEnumerable<MetricKey> metrics, IMetricCalculatorFactory factory, CancellationToken cancellationToken)
```

Reply:
This method is called from the RecalculateMetricStatusesCommand handler, which receives MetricKeys as a string Enumerable. Hence, this change may not be desired.